css-three-column-layout/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <h1>Designed for life science. Optimized for SaMD and Digital Health</h1>
+    <div class="row">
+      <div class="column-full">
```

Comment:
I would recommend setting these to column-third. and then creating a CSS ruleset for it with the proper width, right now flex is just creating its own size for the element and we want to be sure to set it explicitly.